### PR TITLE
add AELIN to token list

### PR DESCRIPTION
**Pull request by alextheboredape**
**Description**
A clear and concise description of the features you're adding in this pull request.
Adding the AELIN token for the Aelin Protocol

**Additional context**
Add any other context about the problem you're solving.

**Metadata**
- Fixes #[Link to Issue]

### CODE REVIEW

**Review by elenadimitrova — CHANGES_REQUESTED** [review]

**Comment by elenadimitrova** [comment]
Also please use a square image of roughly 256x256 pt size, for example the one you use on Coingecko should work.

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1046,6 +1042,17 @@
...
+      "chainId": 10,
+      "address": "0x61BAADcF22d2565B0F471b291C475db5555e0b76",
+      "name": "Aelin Protocol",
+      "symbol": "AELIN",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/AELIN.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
```
[the remark this answers is not in the document]
Comment (0xcdb) [reply optimism.tokenlist.json:1055]: @elenadimitrova Aelin doesn't have a L1 address. Is it mandatory to have one?

[the remark this answers is not in the document]
Comment (elenadimitrova) [reply optimism.tokenlist.json:1055]: If AELIN is L2 native then you don't want to bridge it and should remove the `extensions` section and its `optimismBridgeAddress` entry. See how [sLINK](https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/optimism.tokenlist.json#L396) is setup for example.

[the remark this answers is not in the document]
Comment (0xcdb) [reply optimism.tokenlist.json:1055]: Right, makes sense. Thanks @elenadimitrova

[the remark this answers is not in the document]
Comment (elenadimitrova) [reply optimism.tokenlist.json:1055]: Can you please remove the `"extensions": {` section so we can merge

[the remark this answers is not in the document]
Comment (K-Ho) [reply optimism.tokenlist.json:1055]: Hey @0xcdb bump on removing the `"extensions": {` section so we can merge :)

**Comment by mctopcat** [comment]
Test

File: optimism.tokenlist.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (alextheboredape) [reply optimism.tokenlist.json:1052]: Apologies for the delay. We need to improve our github notifications settings. I have removed the extensions part @elenadimitrova @K-Ho

**Review by Nicca42 — CHANGES_REQUESTED** [review]
Please rebase with master so CI passes, then we can get you merged in 😌 
@0xcdb

**Comment by alextheboredape** [comment]
@Nicca42 @elenadimitrova I have rebased off master and need the workflow to be approved so CI will pass and we can merge this

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**